universe.json

```diff
@@ -0,0 +1,15 @@
+[
+  {
+    "title": "bs-bignumber",
+    "description": "Boilerplate for BuckleScript NPM package",
+    "url": "https://github.com/Astrocoders/bs-universe/tree/master/bs-bignumber"
+  },
+  {
+    "title": "bs-graphql",
+    "url": "https://github.com/Astrocoders/bs-universe/tree/master/bs-graphql"
+  },
+  {
+    "title": "react-native-snap-carousel",
+    "url": "https://github.com/Astrocoders/bs-universe/tree/master/react-native-snap-carousel"
+  }
+]
```

Comment:
There is a circleci job to automate the task of updating this file. It is triggered when there is a merge to the master branch, so I think it is not necessary to update it now

Reply:
Cool!

Reply:
Nice talking about it we can move this CI logic to GitHub actions